CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -105,7 +107,7 @@ class CameraFragment : Fragment() {
     }
 
     /** Blocking camera operations are performed using this executor */
-    private val cameraExecutor = Executors.newSingleThreadExecutor()
+    private lateinit var cameraExecutor: ExecutorService
```

Comment:
Can you change the type to just the generic `Executor`? That way we don't need to add the additional import

Reply:
Ah, makes sense